tests/public-path.test.js

```diff
@@ -15,26 +17,45 @@ const appDir = path.dirname(
   require.resolve('@sku-fixtures/public-path/sku.config.js'),
 );
 
+const distDir = path.resolve(appDir, 'dist');
+
 describe('public path', () => {
-  describe('build and serve', () => {
-    const url = 'http://localhost:4001';
-    const { cancel, signal } = createCancelSignal();
-
-    beforeAll(async () => {
-      await runSkuScriptInDir('build', appDir);
-      runSkuScriptInDir('serve', appDir, { signal });
-      await waitForUrls(url);
-    });
-
-    afterAll(async () => {
-      cancel();
-    });
-
-    it('should create valid app with no unresolved resources', async ({
-      expect,
-    }) => {
-      const app = await getAppSnapshot({ url, expect });
-      expect(app).toMatchSnapshot();
-    });
-  });
+  describe.concurrent.each(['vite', 'webpack'])(
+    'bundler: %s',
+    async (bundler) => {
+      const port = await getPort();
+      const args = ['--strict-port', `--port=${port}`];
+      if (bundler === 'vite')
+        args.push(
+          ...['--experimental-bundler', '--config', 'sku.config.vite.js'],
+        );
+
+      describe('build and serve', () => {
+        const url = `http://localhost:${port}`;
+        const { cancel, signal } = createCancelSignal();
+
+        beforeAll(async () => {
+          await runSkuScriptInDir('build', appDir);
+          runSkuScriptInDir('serve', appDir, { args, signal });
+          await waitForUrls(url);
+        });
+
+        afterAll(async () => {
+          cancel();
+        });
+
+        it('should create valid app with no unresolved resources', async ({
+          expect,
+        }) => {
+          const app = await getAppSnapshot({ url, expect });
+          expect(app).toMatchSnapshot();
+        });
+
+        it('should generate the expected files', async ({ expect }) => {
```

Comment:
This test ensures we snapshot CSS output so we can ensure asset URLs are correct.